windows/README.md

```diff
@@ -0,0 +1,53 @@
+## Bahmni Windows Service
+=============================
+
+### Background
+The purpose of the Bahmni Service is to automatically start and gracefully halt a Vagrant Virtual Machine (VM). 
+
+The service operates as follows:
+
+1. At configured intervals, checks to see if the VM is running
+1. If the VM is not running, start the VM
+1. If the VM is running and the service is stopped (user action or when the hosting windows machine has been shutdown), the service will gracefully shutdown the VM
+
+The windows service is configured to automatically start whenever the windows server is booted up. The deafult interval for checking the status of the VM is 10mins.
+
+### MSI Configuration Steps
+Assuming that VirtualBox is already installed on a 64bit Windows Professional machine together with the Bahmni Vagrant CentOS instance, please follow these steps to install and configure the Bahmni Windows Service:
+
+1. Download and install the [Bahmni Service MSI](https://github.com/jembi/cameroon-bahmni-config/blob/COM-823/windows/setup/Bahmni%20Service.msi) on your windows server.
```

Comment:
@rmrlangford We will upload the Bahmni Service.msi on the release page instead as this branch will be deleted. Please use this path instead: https://github.com/jembi/cameroon-openmrs-module-bahmniapps/releases/download/v1.5.0/Bahmni.Service.msi

Reply:
@lauravignoli , done